### ISSUE TITLE
Title of page changes, but correct page not shown.

### ISSUE THREAD

**Issue by ncs-git** [deniaz/wp-dms#7]
Good day.

Love the simplicity of this plugin. However, after installation and setup, the title of the page changes in the browser, however, the content of the page is the default homepage of the apachevhost instead of the page that is mapped. 

www.accglending.com 

The above domain should show the page www.arcapital.com/acc. However it simply displays the content from here www.arcapital.com You notice the page title changes to say "Lending" which is correct, but it doesn't actually redirect to the page. 

Thoughts? 

Have a very blessed day, 
Damon

**Comment by deniaz** [comment]
Hi Damon

Thanks for the flowers ;)

I have a what it could be. I'll look into it over the weekend, if that's alright with you?

Regards,
Rob

**Comment by ncs-git** [comment]
Rob --

Thank you so much for your prompt reply. This weekend would be just fine. No worries. I greatly appreciate your help.

Have a very blessed day,
Damon

**Comment by deniaz** [comment]
Hi Damon

Quick questions; do you happen to use a "static" homepage?  (Settings > Read)

Regards,
Rob

**Comment by ncs-git** [comment]
Yes sir. I am fairly certain that it is static. I will double check in just a minute for you when I get back to a computer. Thank you sir. Have a blessed day. 

Damon's Brevity disclaimer: Sent from my iPhone

**Comment by ncs-git** [comment]
Yes sir. The home page is a static page. Have a blessed day. 

Damon's Brevity disclaimer: Sent from my iPhone

**Comment by ncs-git** [comment]
Morning Rob. Just checking on this issue. Didn't know if you had time to devote to it yet or not. Thank you again for looking into the issue.

Have a very blessed day,
Damon

**Comment by deniaz** [comment]
Good Morning Damon,

I'm quite busy right now, sorry about that. I started looking into the issue, and I'm not quite sure where it happens. I've tried it with a static homepage, but that doesn't seem to be the issue. I'm still investigating this.

What version of WordPress are you using?

**Comment by ncs-git** [comment]
Good morning Robert.

We are using WP v 3.5.1 (I am going to upgrade to 3.5.2 this morning however). Hopefully that helps. Thanks again for working on this.

Have a very blessed day,
Damon Hill
